docs/src/main/paradox/server/index.md

```diff
@@ -9,6 +9,7 @@ Get our Quickstart project (available for [Scala](https://developer.lightbend.co
  * [Walkthrough](walkthrough.md)
  * [Status codes](statuscodes.md)
  * [gRPC-Web](grpc-web.md)
+ * [Failures](failures.md)
```

Comment:
Is `failures` missing? Jenkins job fails because of that.

```
[error] com.lightbend.paradox.markdown.Index$LinkException: Unknown page [failures.md] linked from [server/index.md]
[error] 	at com.lightbend.paradox.markdown.Index$.$anonfun$links$2(Index.scala:191)
[error] 	at scala.Option.getOrElse(Option.scala:189)
[error] 	at com.lightbend.paradox.markdown.Index$.lookup$1(Index.scala:191)
[error] 	at com.lightbend.paradox.markdown.Index$.$anonfun$links$3(Index.scala:198)
[error] 	at com.lightbend.paradox.markdown.Index$.$anonfun$links$3$adapted(Index.scala:197)
[error] 	at scala.collection.immutable.List.foreach(List.scala:392)
[error] 	at com.lightbend.paradox.markdown.Index$.add$1(Index.scala:197)
```